content/Store Support/Payments and Invoices/Remove-payment-information-from-your-Arduino-account.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Remove payment information from your Arduino account"
+---
+
+1. Navigate to [Arduino.cc](https://www.arduino.cc/) and ensure that you're logged into your Arduino account.
+
+1. Click on your profile icon located on the top right of the screen
```

Comment:
Yes, if a profile picture is associated to the account, either via Arduino preferences or SSO, it will be displayed instead of the default symbol. That's a good point.

In the context of Cloud, we also have the problem on app.arduino.cc pages, the proflie menu is displayed on the top-**right** instead of the top-left.

![image](https://github.com/user-attachments/assets/24c2c05d-054d-442f-bc62-9837d2595cc9)

Perhaps "click on your profile button in the top navigation" is enough here.